Prompt has spill over at the top on every theme

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

Hi The prompts appear to have a spill over of colour. This shows on every theme if i also run get-poshthemes

Like this

![image](https://user-images.githubusercontent.com/662108/223463942-cd2ff6b8-51e7-4ba6-a4d2-051af13ab40b.png)

Using the latest windows terminal and powershell 

### Theme

default but all of them

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
Version: 14.12.0

Segments:

ConsoleTitle(true) -   7 ms - pwsh in User1
session(true)      -   2 ms -  User1 
path(true)         -   1 ms -   ~
git(false)         -  11 ms -
root(false)        -   0 ms -
exit(true)         -   0 ms -   
node(false)        -   0 ms -
go(false)          -   0 ms -
python(false)      -   0 ms -
shell(true)        -   0 ms - in pwsh
time(true)         -   0 ms - at 15:15:37

Run duration: 28.4269ms

Cache path: C:\Users\User1\AppData\Local\oh-my-posh

Config path:

Logs:

[DEBUG] 15:15:37.728 shell.go:Getenv:381 → C:\Users\User1\AppData\Local
[TRACE] 15:15:37.728 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 15:15:37.729 shell_windows.go:CachePath() - 599.6µs
[DEBUG] 15:15:37.731 shell.go:Getenv:381 → NO DATA
[TRACE] 15:15:37.731 shell.go:Getenv(POSH_THEME) - 0s
[DEBUG] 15:15:37.731 shell.go:resolveConfigPath:313 → No config set, fallback to default config
[TRACE] 15:15:37.731 shell.go:resolveConfigPath() - 0s
[TRACE] 15:15:37.731 shell.go:Init() - 2.972ms
[TRACE] 15:15:37.731 shell.go:Flags() - 0s
[DEBUG] 15:15:37.731 config.go:loadConfig:105 → no config file specified, using default
[TRACE] 15:15:37.731 config.go:loadConfig() - 0s
[TRACE] 15:15:37.732 shell.go:Flags() - 0s
[DEBUG] 15:15:37.732 shell.go:Getenv:381 → NO DATA
[TRACE] 15:15:37.732 shell.go:Getenv(OMP_CACHE_DISABLED) - 0s
[TRACE] 15:15:37.732 shell_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[TRACE] 15:15:37.732 shell.go:GOOS() - 0s
[TRACE] 15:15:37.732 shell.go:GOOS() - 0s
[TRACE] 15:15:37.732 shell.go:GOOS() - 0s
[TRACE] 15:15:37.732 shell.go:GOOS() - 0s
[TRACE] 15:15:37.732 shell.go:GOOS() - 0s
[TRACE] 15:15:37.732 shell.go:GOOS() - 0s
[TRACE] 15:15:37.732 shell.go:GOOS() - 0s
[TRACE] 15:15:37.732 shell.go:GOOS() - 0s
[TRACE] 15:15:37.732 shell.go:GOOS() - 0s
[TRACE] 15:15:37.732 shell.go:GOOS() - 0s
[TRACE] 15:15:37.732 shell.go:GOOS() - 0s
[TRACE] 15:15:37.732 shell.go:GOOS() - 0s
[TRACE] 15:15:37.732 shell.go:GOOS() - 0s
[TRACE] 15:15:37.732 shell.go:GOOS() - 0s
[TRACE] 15:15:37.732 shell.go:GOOS() - 0s
[TRACE] 15:15:37.732 shell.go:GOOS() - 0s
[TRACE] 15:15:37.732 shell.go:GOOS() - 0s
[TRACE] 15:15:37.732 shell.go:GOOS() - 0s
[TRACE] 15:15:37.732 shell.go:GOOS() - 0s
[DEBUG] 15:15:37.732 shell_windows.go:WindowsRegistryKeyValue:206 → ColorizationColor(DWORD): 0xC40099BC
[TRACE] 15:15:37.732 shell_windows.go:Root() - 0s
[DEBUG] 15:15:37.739 shell.go:Shell:626 → process name: pwsh.exe
[TRACE] 15:15:37.739 shell.go:Shell() - 7.15ms
[TRACE] 15:15:37.739 shell.go:ErrorCode() - 0s
[TRACE] 15:15:37.739 shell_windows.go:IsWsl() - 0s
[TRACE] 15:15:37.739 shell.go:GOOS() - 0s
[DEBUG] 15:15:37.739 shell.go:Pwd:409 → NO DATA
[TRACE] 15:15:37.739 shell.go:Pwd() - 0s
[DEBUG] 15:15:37.739 shell_windows.go:56 → C:\Users\User1
[TRACE] 15:15:37.739 shell.go:GOOS() - 0s
[TRACE] 15:15:37.739 shell.go:GOOS() - 0s
[TRACE] 15:15:37.739 shell.go:GOOS() - 0s
[TRACE] 15:15:37.739 shell.go:GOOS() - 0s
[TRACE] 15:15:37.739 shell.go:GOOS() - 0s
[TRACE] 15:15:37.739 shell.go:GOOS() - 0s
[TRACE] 15:15:37.739 shell.go:GOOS() - 0s
[TRACE] 15:15:37.739 shell.go:GOOS() - 0s
[TRACE] 15:15:37.739 shell.go:GOOS() - 0s
[DEBUG] 15:15:37.739 shell.go:User:530 → User1
[TRACE] 15:15:37.739 shell.go:User() - 0s
[DEBUG] 15:15:37.739 shell.go:Host:542 → A33101
[TRACE] 15:15:37.739 shell.go:Host() - 0s
[TRACE] 15:15:37.739 shell.go:GOOS() - 0s
[TRACE] 15:15:37.739 shell.go:TemplateCache() - 7.15ms
[TRACE] 15:15:37.739 shell.go:Flags() - 0s
[DEBUG] 15:15:37.739 shell.go:Pwd:389 → C:\Users\User1
[TRACE] 15:15:37.739 shell.go:Pwd() - 0s
[DEBUG] 15:15:37.739 shell_windows.go:56 → C:\Users\User1
[TRACE] 15:15:37.739 shell.go:GOOS() - 0s
[DEBUG] 15:15:37.739 shell.go:Getenv:381 → NO DATA
[TRACE] 15:15:37.739 shell.go:Getenv(SSH_CONNECTION) - 0s
[DEBUG] 15:15:37.739 shell.go:Getenv:381 → NO DATA
[TRACE] 15:15:37.739 shell.go:Getenv(SSH_CLIENT) - 0s
[TRACE] 15:15:37.739 shell.go:TemplateCache() - 0s
[TRACE] 15:15:37.742 shell.go:TemplateCache() - 0s
[TRACE] 15:15:37.742 shell.go:Shell() - 0s
[DEBUG] 15:15:37.742 shell.go:Pwd:389 → C:\Users\User1
[TRACE] 15:15:37.742 shell.go:Pwd() - 0s
[DEBUG] 15:15:37.742 shell_windows.go:56 → C:\Users\User1
[TRACE] 15:15:37.742 shell.go:GOOS() - 0s
[DEBUG] 15:15:37.742 shell.go:Pwd:389 → C:\Users\User1
[TRACE] 15:15:37.742 shell.go:Pwd() - 0s
[TRACE] 15:15:37.742 shell.go:Shell() - 0s
[TRACE] 15:15:37.742 shell.go:Flags() - 0s
[DEBUG] 15:15:37.742 shell_windows.go:56 → C:\Users\User1
[TRACE] 15:15:37.742 shell.go:GOOS() - 0s
[TRACE] 15:15:37.742 shell.go:GOOS() - 0s
[TRACE] 15:15:37.742 shell.go:PathSeparator() - 93µs
[TRACE] 15:15:37.742 shell.go:GOOS() - 0s
[TRACE] 15:15:37.742 shell.go:GOOS() - 0s
[TRACE] 15:15:37.742 shell.go:GOOS() - 0s
[TRACE] 15:15:37.742 shell.go:GOOS() - 0s
[TRACE] 15:15:37.742 shell.go:GOOS() - 0s
[TRACE] 15:15:37.742 shell.go:GOOS() - 0s
[TRACE] 15:15:37.742 shell.go:GOOS() - 0s
[TRACE] 15:15:37.742 shell.go:PathSeparator() - 0s
[TRACE] 15:15:37.742 shell.go:PathSeparator() - 0s
[TRACE] 15:15:37.742 shell.go:GOOS() - 0s
[TRACE] 15:15:37.742 shell.go:GOOS() - 0s
[TRACE] 15:15:37.742 shell.go:GOOS() - 0s
[TRACE] 15:15:37.742 shell.go:PathSeparator() - 0s
[TRACE] 15:15:37.742 shell.go:GOOS() - 0s
[TRACE] 15:15:37.742 shell.go:GOOS() - 0s
[TRACE] 15:15:37.742 shell.go:GOOS() - 0s
[TRACE] 15:15:37.742 shell.go:PathSeparator() - 0s
[TRACE] 15:15:37.742 shell.go:GOOS() - 0s
[TRACE] 15:15:37.742 shell.go:GOOS() - 0s
[TRACE] 15:15:37.742 shell.go:GOOS() - 0s
[TRACE] 15:15:37.742 shell.go:PathSeparator() - 0s
[DEBUG] 15:15:37.742 shell.go:Pwd:389 → C:\Users\User1
[TRACE] 15:15:37.742 shell.go:Pwd() - 0s
[TRACE] 15:15:37.742 shell_windows.go:IsWsl() - 0s
[TRACE] 15:15:37.742 shell.go:StackCount() - 0s
[DEBUG] 15:15:37.743 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 15:15:37.743 win32_windows.go:isWriteable:303 → current user is member of S-1-5-32-544
[DEBUG] 15:15:37.743 win32_windows.go:isWriteable:311 ↓
    WRITE_DAC
    WRITE_OWNER
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 15:15:37.744 win32_windows.go:isWriteable:313 → user has write access
[TRACE] 15:15:37.744 shell_windows.go:DirIsWritable() - 1.6422ms
[TRACE] 15:15:37.744 shell.go:TemplateCache() - 0s
[TRACE] 15:15:37.744 shell.go:TemplateCache() - 0s
[TRACE] 15:15:37.744 shell.go:Shell() - 0s
[DEBUG] 15:15:37.744 shell.go:Pwd:389 → C:\Users\User1
[TRACE] 15:15:37.744 shell.go:Pwd() - 0s
[DEBUG] 15:15:37.744 shell_windows.go:56 → C:\Users\User1
[TRACE] 15:15:37.744 shell.go:GOOS() - 0s
[TRACE] 15:15:37.744 shell.go:GOOS() - 0s
[DEBUG] 15:15:37.755 shell.go:CommandPath:581 → C:\Program Files\Git\cmd\git.exe
[TRACE] 15:15:37.755 shell.go:CommandPath(git.exe) - 11.4137ms
[TRACE] 15:15:37.755 shell.go:HasCommand(git.exe) - 11.5318ms
[DEBUG] 15:15:37.755 shell.go:Pwd:389 → C:\Users\User1
[TRACE] 15:15:37.755 shell.go:Pwd() - 0s
[ERROR] 15:15:37.756 shell.go:HasParentFilePath:714 → CreateFile .git: The system cannot find the file specified.
[TRACE] 15:15:37.756 shell.go:HasParentFilePath(.git) - 416.2µs
[DEBUG] 15:15:37.756 shell.go:Pwd:389 → C:\Users\User1
[TRACE] 15:15:37.756 shell.go:Pwd() - 0s
[DEBUG] 15:15:37.756 shell_windows.go:56 → C:\Users\User1
[TRACE] 15:15:37.756 shell.go:GOOS() - 0s
[TRACE] 15:15:37.756 shell_windows.go:Root() - 132.1µs
[DEBUG] 15:15:37.756 shell.go:Pwd:389 → C:\Users\User1
[TRACE] 15:15:37.756 shell.go:Pwd() - 0s
[DEBUG] 15:15:37.756 shell_windows.go:56 → C:\Users\User1
[TRACE] 15:15:37.756 shell.go:GOOS() - 0s
[TRACE] 15:15:37.756 shell.go:ErrorCode() - 0s
[TRACE] 15:15:37.756 shell.go:TemplateCache() - 0s
[TRACE] 15:15:37.756 shell.go:TemplateCache() - 0s
[TRACE] 15:15:37.756 shell.go:Shell() - 0s
[TRACE] 15:15:37.756 shell.go:TemplateCache() - 0s
[TRACE] 15:15:37.756 shell.go:Flags() - 0s
[DEBUG] 15:15:37.756 shell.go:Pwd:389 → C:\Users\User1
[TRACE] 15:15:37.756 shell.go:Pwd() - 0s
[DEBUG] 15:15:37.756 shell_windows.go:56 → C:\Users\User1
[TRACE] 15:15:37.756 shell.go:GOOS() - 0s
[DEBUG] 15:15:37.756 shell.go:Pwd:389 → C:\Users\User1
[TRACE] 15:15:37.756 shell.go:Pwd() - 0s
[DEBUG] 15:15:37.756 shell_windows.go:56 → C:\Users\User1
[DEBUG] 15:15:37.756 shell.go:Pwd:389 → C:\Users\User1
[TRACE] 15:15:37.756 shell.go:Pwd() - 0s
[DEBUG] 15:15:37.756 shell_windows.go:56 → C:\Users\User1
[TRACE] 15:15:37.756 shell.go:GOOS() - 0s
[DEBUG] 15:15:37.756 shell.go:Pwd:389 → C:\Users\User1
[TRACE] 15:15:37.756 shell.go:Pwd() - 0s
[DEBUG] 15:15:37.756 shell_windows.go:56 → C:\Users\User1
[DEBUG] 15:15:37.756 shell.go:Pwd:389 → C:\Users\User1
[TRACE] 15:15:37.756 shell.go:Pwd() - 0s
[DEBUG] 15:15:37.756 shell_windows.go:56 → C:\Users\User1
[TRACE] 15:15:37.756 shell.go:GOOS() - 0s
[DEBUG] 15:15:37.756 shell.go:Pwd:389 → C:\Users\User1
[TRACE] 15:15:37.756 shell.go:Pwd() - 0s
[DEBUG] 15:15:37.756 shell_windows.go:56 → C:\Users\User1
[DEBUG] 15:15:37.756 shell.go:Pwd:389 → C:\Users\User1
[TRACE] 15:15:37.756 shell.go:Pwd() - 0s
[DEBUG] 15:15:37.756 shell_windows.go:56 → C:\Users\User1
[TRACE] 15:15:37.756 shell.go:GOOS() - 0s
[TRACE] 15:15:37.756 shell.go:Shell() - 0s
[TRACE] 15:15:37.756 shell.go:Flags() - 0s
[TRACE] 15:15:37.756 shell.go:TemplateCache() - 0s
[TRACE] 15:15:37.757 shell.go:TemplateCache() - 0s
[TRACE] 15:15:37.757 shell.go:Shell() - 0s
[DEBUG] 15:15:37.757 shell.go:Pwd:389 → C:\Users\User1
[TRACE] 15:15:37.757 shell.go:Pwd() - 0s
[DEBUG] 15:15:37.757 shell_windows.go:56 → C:\Users\User1
[TRACE] 15:15:37.757 shell.go:GOOS() - 0s
[TRACE] 15:15:37.757 shell.go:TemplateCache() - 0s
[TRACE] 15:15:37.757 shell.go:TemplateCache() - 0s
[TRACE] 15:15:37.757 shell.go:Shell() - 0s
[DEBUG] 15:15:37.757 shell.go:Getenv:381 → C:\Users\User1\AppData\Local
[TRACE] 15:15:37.757 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 15:15:37.757 shell_windows.go:CachePath() - 65.2µs
[TRACE] 15:15:37.757 shell.go:Flags() - 0s
```

--- a/website/docs/installation/fonts.mdx
+++ b/website/docs/installation/fonts.mdx
@@ -14,7 +14,7 @@ We recommend [Meslo LGM NF][meslo], but any Nerd Font should be compatible with 
 
 To see the icons displayed in Oh My Posh, **install** a [Nerd Font][nerdfonts], and **configure** your terminal to use it.
 
-#### Installation
+### Installation
 
 <Tabs
     defaultValue="oh-my-posh"
@@ -51,7 +51,23 @@ brew install --cask font-fira-code-nerd-font
 </TabItem>
 </Tabs>
 
-#### Windows Terminal
+### Configuration
+
+Make sure to **configure your terminal** to use the font you have installed. The following sections will show you how to do this for the most popular terminals.
+
+<Tabs
+    defaultValue="wt"
+    values={[
+        { label: 'Windows Terminal', value: 'wt' },
+        { label: 'VS Code', value: 'code' }
+    ]
+}>
+<TabItem value="wt">
+
+:::caution Text renderer
+To ensure correct rendering of the glyphs you will need to enable the option `Use the new text renderer ("AtlasEngine")` in your terminal settings.
+For further details, see [here][wt-issue-8993].
+:::
 
 Once you have installed a Nerd Font, you will need to configure the Windows Terminal to use it. This can be easily done
 by modifying the Windows Terminal settings (default shortcut: `CTRL + SHIFT + ,`). In your `settings.json` file, add the
@@ -72,7 +88,8 @@ by modifying the Windows Terminal settings (default shortcut: `CTRL + SHIFT + ,`
 }
 ```
 
-#### VS Code
+</TabItem>
+<TabItem value="code">
 
 When using Visual Studio Code, you will need to configure the integrated Terminal to make use of the Nerd Font as well. This can be done by changing the `Integrated: Font Family` value in the Terminal settings (default shortcut: `CTRL + ,` and search for `Integrated: Font Family` or via `Users` -> `Features` -> `Terminal`).
 
@@ -81,6 +98,9 @@ If you are using the JSON based settings, you will need to update the `terminal.
 ```json
 "terminal.integrated.fontFamily": "MesloLGM NF"
 ```
+
+</TabItem>
+</Tabs>
 
 ### Other Fonts
 
@@ -94,3 +114,4 @@ The `minimal` themes do not make use of Nerd Font icons.
 [themes]: https://github.com/JanDeDobbeleer/oh-my-posh/tree/main/themes
 [font-thread]: https://github.com/JanDeDobbeleer/oh-my-posh/issues/145#issuecomment-730162622
 [configuration]: /docs/configuration/overview
+[wt-issue-8993]: https://github.com/microsoft/terminal/issues/8993

